test/alloy/TestAlloyUtils.java

```diff
@@ -180,20 +180,20 @@ public void testGetPathPredicate() {
         SigData sigData = new SigData(createNewSig());
         String expected = String.join("\n",
             "",
-            "pred path_c0[s: A] {",
+            "pred state_s0[s: A] {",
```

Comment:
This should probably be `state_s1` since we don't place any restrictions on the first state (`s0`).

You can change `i` to `i+1` [here](https://github.com/WatForm/aldb/blob/master/src/alloy/AlloyUtils.java#L128).